Podcast/FollowerFollowingViewController.swift

```diff
@@ -115,15 +101,43 @@ class FollowerFollowingViewController: ViewController, UITableViewDataSource, UI
         cell.delegate = self
         return cell
     }
-    
+
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         return SearchPeopleTableViewCell.cellHeight
     }
-    
+
+}
+
+// MARK: TableView Delegate
+
+extension FollowerFollowingViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let user = users[indexPath.row]
+        let profileViewController = UserDetailViewController(user: user)
+        navigationController?.pushViewController(profileViewController, animated: true)
+    }
+}
+
+// MARK: SearchPeopleTableViewCell Delegate
+
+extension FollowerFollowingViewController: SearchPeopleTableViewCellDelegate {
     func searchPeopleTableViewCellDidPressFollowButton(cell: SearchPeopleTableViewCell) {
         guard let indexPath = usersTableView.indexPath(for: cell) else { return }
         let user = users[indexPath.row]
         user.followChange(completion: cell.setFollowButtonState)
     }
+}
+
+// MARK: EmptyStateTableView Delegate
+
+extension FollowerFollowingViewController: EmptyStateTableViewDelegate {
+
+    func emptyStateTableViewHandleRefresh() {
+        fetchUsers()
+    }
+
+    func didPressEmptyStateViewActionItem() {
```

Comment:
make this all one line if nothing in it and remove comment

Reply:
it's a required method in the protocol so i think that's why the comment is there to specify why it's a noop